src/main/java/frc/robot/RobotContainer.java

```diff
@@ -98,7 +98,8 @@ private void configureButtonBindings() {
             double xPower = -driveController.getLeftY();
             double yPower = -driveController.getLeftX();
             double angularPower = -driveController.getRightX();
-            swerveSubsystem.setSwerveDrivePowers(xPower, yPower, angularPower);
+            boolean relative = driveController.getRightTriggerAxis() > 0.75;
+            swerveSubsystem.setSwerveDrivePowers(xPower, yPower, angularPower, relative);
```

Comment:
this'll be pretty awkward with rishay's abstractions.. perhaps in this section we can just use an if to cast to Tank or Swerve and just have all the setting logic separate.